src/ensembl/src/content/app/species/SpeciesPageContent.tsx

```diff
@@ -59,7 +59,7 @@ const SpeciesPageContent = () => {
   }, [genomeId]);
 
   return (
-    <>
+    <div className={styles.speciesPage}>
```

Comment:
Do we need to add any style/class to it, just a wrapper div will be enough to make it obey the parent grid layout. will the height 100% makes a difference when there is less content. maybe we should have a min-height instead of 100%